mochitest.sh

```diff
@@ -15,9 +15,17 @@ cd /activity-stream && npm install . && npm run buildmc
 cd /mozilla-central && ./mach build \
   && ./mach test browser_parsable_css \
   && ./mach lint -l codespell browser/components/newtab \
-  && ./mach test browser/components/newtab/test/browser --headless \
-  && ./mach test browser/components/newtab/test/xpcshell \
-  && ./mach test browser/components/preferences/in-content/tests/browser_hometab_restore_defaults.js --headless \
-  && ./mach test browser/components/preferences/in-content/tests/browser_newtab_menu.js --headless \
-  && ./mach test browser/components/enterprisepolicies/tests/browser/browser_policy_set_homepage.js --headless \
-  && ./mach test browser/components/preferences/in-content/tests/browser_search_subdialogs_within_preferences_1.js --headless
+  && ./mach test browser/components/newtab/test/browser \
```

Comment:
Looks like mochitests failing have exit code 0.. probably file a bug for this

Reply:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1483003

If it gets fixed, we could get rid of the grepping and just keep all the `./mach test … … …` in a single command, as that prints the summary together as well as running all the tests.